exonum/src/storage/tests.rs

```diff
@@ -259,3 +263,18 @@ mod rocksdb_tests {
         assert_eq!(index.len() as usize, iter.count());
     }
 }
+
+#[test]
+#[ignore]
+#[allow(unreachable_code, unused_variables)]
+fn should_compile() {
```

Comment:
@blackbeam i'd prefer to see real tests for covering this feature.

Reply:
I suppose a compile-time test is OK, but I would add a comment because `#[ignore]` attribute is too generic.

Reply:
Looks like it doesn't have to be a test.
Will do something similar to this: https://github.com/rust-lang/rust/pull/34951/files

Reply:
Actually, I think the current solution is better than `dead_code` function, because it will not affect compile time.

Reply:
@vitvakatu, it's in separete test module annotated with `#[test]` so compile time won't be affected 🙃